CiTo.cs

```diff
@@ -82,7 +83,10 @@ public static int Main(string[] args)
 					Console.WriteLine("cito 1.0.0");
 					return 0;
 				case "-l":
-					lang = args[++i];
+					while (!args[i + 1].StartsWith('-')){
```

Comment:
1. This will eat source filenames as languages if you use:

    cito -l c cpp foo.ci

2. `IndexOutOfRangeException` unless an option follows.

I like the idea, but would prefer to use comma-separated syntax:

    cito -l c,cpp,java -o foo.c foo.ci

Reply:
Yeah that was just a quick and dirty solution from me. I'm still not happy that you have to give a `-o foo.c` but generate a `foo.cpp` and `foo.cs` etc. Maybe there ist a even better solution? Something like [wax ](https://github.com/LingDong-/wax ) does: `--c path/out.c --java  path/ --cs path/out.cs ` etc...